[Docs] Fix merge errors / start vale checking in docs (#422)
mckinsey/vizro: vizro-core/docs/pages/user-guides/card-button.md

    [guide on navigation](navigation.md).

A navigation card enables you to navigate to a different page via a click on the card area.
To create a navigation card, do the following:

- Insert the [`Card`][vizro.models.Card] into the `components` argument of the [`Page`][vizro.models.Page]
- Give the `text` parameter with a title and some description

> - Give the `href` parameter (relative or absolute URL)

Reviewer: **Provide** sounds better here - doesn't it?

"Give the `text` parameter with a title and some description" sounds a bit odd to me 😄

> - Give the `text` parameter with a title and some description
> - Give the `href` parameter (relative or absolute URL)

Author: ```suggestion
- The `text` parameter contains a title and some description.
- The `href` parameter is for a relative or absolute URL.
```

> - The `text` parameter contains a title and some description.
> - The `href` parameter is for a relative or absolute URL.

Reviewer: I think this might have worked better before. It probably didn't become very clear but this should be instructions of how to create the navigation card so:

1. Do this
2. Then do this
3. Do this

It doesn't have to be in that order though. So I think it should start with a verb - WDYT @stichbury ?

Maybe:

```suggestion
- Pass your markdown text to the `Card.text`
- Pass a relative or absolute URL to the `Card.href`
```
